extensions/mongodb-client/runtime/src/test/java/io/quarkus/mongodb/MongoWithReplicasTestBase.java

```diff
@@ -139,15 +138,6 @@ private static boolean isReplicaSetStarted(final Document setting) {
         return true;
     }
 
-    private List<IMongodConfig> buildMongodConfiguration(Map<String, Integer> urls, final boolean configureReplicaSet)
-            throws IOException {
-        final List<IMongodConfig> configs = new ArrayList<>(urls.size());
-        for (Map.Entry<String, Integer> url : urls.entrySet()) {
-            configs.add(buildMongodConfiguration(url.getKey(), url.getValue(), configureReplicaSet));
-        }
-        return configs;
-    }
```

Comment:
I let @cescoffier comment on this one. Maybe it should have been called from somewhere.

Reply:
No, it can be removed. It was one of the approaches I tried to get replicas working...